jetty-server/src/main/java/org/eclipse/jetty/server/HttpChannelState.java

```diff
@@ -325,6 +326,7 @@ public void startAsync(AsyncContextEvent event)
             _event = event;
             lastAsyncListeners = _asyncListeners;
             _asyncListeners = null;
+            _ignoreComplete = false;
```

Comment:
This is unnecessary, as by default booleans are set to `false`. What's missing is to reset this field in `recycle()`!

Reply:
it is reset in startAsync.... but doing so in reset as well will be good.

Reply:
ah this is startAsync.  Is needed here for multiple async cycles... but also good to have in reset.